packages/sdk/docs/modules.md

```diff
@@ -0,0 +1,850 @@
+[@eth-optimism/sdk](README.md) / Exports
```

Comment:
This should probably be the "homepage" for SDK docs rather than a separate README. Also this should be the page we link to from community hub